lib/activejob/lockable/lockable.rb

```diff
@@ -58,7 +65,11 @@ def locked_ttl
       private
 
       def lock_period
-        options[:lock]
+        options&.fetch(:lock, 0).to_i
```

Comment:
This is quite cumbersome to read, I think we might be better extending it to a few lines

```ruby
def lock_period
  return 0 unless options.key?(:lock)

  options[:lock].to_i
end
```

wdyt?